devops/arangodb.yml

```diff
@@ -42,11 +47,39 @@
       shell: ARANGODB_DEFAULT_ROOT_PASSWORD={{ arangodb_root_password }} arango-secure-installation
       become: true
 
+    - name: Copy the compressed ssl certificate files to the server
+      copy:
+        src: '{{ ssl_cert_path }}'
+        dest: /home/ubuntu/
+        follow: yes
+        mode: u=rw,g=,o=
+
+    - name: Combine the chain and the private key into one file (required by arango)
+      shell:
+        chdir: /home/ubuntu/
+        cmd: cat fullchain.pem privkey.pem > server.pem
+
+    - name: Set the combined file as 600 permission
+      file:
+        path: /home/ubuntu/server.pem
+        mode: u=rw,g=,o=
+
+    - name: Add an exception to the permissions for the arangodb user
+      shell:
+        chdir: /home/ubuntu/
+        cmd: setfacl -m u:arangodb:r /home/ubuntu/server.pem
```

Comment:
Why do we need an exception for this user? Where does the `setfacl` command come from?

Reply:
We need an exception for this user so that arango can read the key file. The other option would be to set arangodb as the owner of the file but I'm not sure that's best. The attack vector is that someone has access to the arangodb user and could modify the SSL cert files and setting arangodb as the owner would allow an attacker to change the permissions on the file to `+w`.

The `setfacl` command is just a linux utility for setting file permissions in a more granular way using ACLs (access control lists). I've seen CHPC used it in production here at the U in the protected environment, so I'm fairly certain that this is decent way to do it. I guess the other option is to add a new group, add arangodb to that group, and then make that group the owner of the file. I think ACLs are a faster and cleaner way of managing the permission

Reply:
Yeah this should be ok. If you have any reading you've done on ACLs, it would be interesting to see those. I don't know much about them, so thanks for bringing them in here.